test/fixtures/gcs-fixture/build.gradle

```diff
@@ -9,7 +9,6 @@
 apply plugin: 'elasticsearch.java'
 
 description = 'Fixture for Google Cloud Storage service'
-tasks.named("test").configure { enabled = false }
```

Comment:
For some reason tests were disabled? (if I'm reading the gradle correctly)

Reply:
Probably the same sort of reason that all tests were disabled in `:test:fixtures`. See https://github.com/elastic/elasticsearch/pull/117329. Whatever the reason was, it's no longer valid for sure.